data/campaigns/Legend_of_Wesmere/scenarios/chapter1/01_The_Uprooting.cfg

```diff
@@ -357,7 +357,7 @@ This is the story of Kalenz, Landar, and of the Elves in the first days of the h
         [/message]
         [message]
             id=Kalenz
-            message= _ "We must reach the elvish council in Ka’lian and enlist their help to recapture our home."
+            message= _ "We must reach the Elvish Council in Ka’lian and enlist their help to recapture our home."
```

Comment:
Why capitalize this one?

Reply:
Precedents:
https://github.com/wesnoth/wesnoth/blob/6c78d376fe8f4d6b62d6392bef738e500854851d/data/campaigns/Legend_of_Wesmere/scenarios/chapter4/14_Human_Alliance.cfg#L367
and
https://github.com/wesnoth/wesnoth/blob/6c78d376fe8f4d6b62d6392bef738e500854851d/data/campaigns/Legend_of_Wesmere/scenarios/chapter5/24_Epilogue.cfg#L25

Though I grant that the instances of lower case far outweigh the upper case, but as mentioned in the original submission I consider capitalisation appropriate when referring to a specific Council, as opposed to any generic council.

Reply:
Alright. Personally I just think it looks clunky, but it's not a big deal.